src/fty_discovery_server.cc

```diff
@@ -687,7 +694,16 @@ void ftydiscovery_create_asset(fty_discovery_server_t* self, zmsg_t** msg_p)
                     link.dest = fty::convert<uint32_t>(DBAssets::name_to_asset_id(iname));
                 }
                 auto conn = tntdb::connectCached(DBConn::url);
-                DBAssetsInsert::insert_into_asset_links(conn, self->default_values_links);
+                // In device centric view for sts, add default link for the 2 inputs if default links have only one entry
```

Comment:
we have memleaks in the two return blocks L686 L678    (missing zmsg_destroy(&response) )

Reply:
Well seen I will fix them